wazo_confd/plugins/voicemail/validator.py

```diff
@@ -48,6 +60,14 @@ def validate(self, voicemail):
             )
 
 
+class AssociatedToTenant(Validator):
```

Comment:
```suggestion
class PreventSharedVoicemailUserAssociation(Validator):
```
Suggestion is not much better, but to me the current name is not representative of behavior - I'd like something a bit more explicit

Again, no hard opinion here